src/components/ReviewSection.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+  Text,
+  Grid,
+  GridItem,
+  TextListItem,
+  TextVariants,
+  TextListItemVariants,
+} from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+
+const ReviewSection = ({ title, data, testid }) => {
+  return (
+    <Grid data-testid={testid} hasGutter>
+      <GridItem span={12} hasGutter>
+        <Text component={TextVariants.h1}>{title}</Text>
+      </GridItem>
+      {data.map(({ name, value }) => (
+        <>
+          <GridItem span={3} hasGutter>
+            <TextListItem component={TextListItemVariants.dt}>
+              {name}
+            </TextListItem>
+          </GridItem>
+          <GridItem style={{ paddingBottom: '20px' }} span={9} hasGutter>
```

Comment:
The padding here makes the review looks too spacey. Can you remove this?

Reply:
In a second though change it to `1rem` or `16px` + add this as a class in App.scss